code/modules/holiday/halloween.dm

```diff
@@ -45,8 +45,8 @@
 /obj/joeq/spooky
 	name = "Memorial Plaque"
 
-	examine(mob/user)
-		boutput(usr, "Here lies [user.real_name]. Loved by all. R.I.P.")
+	get_desc(dist, mob/user)
+		. += "Here lies [user.real_name]. Loved by all. R.I.P."
```

Comment:
Possibly should be a separate commit outside the PR for cleaner organisation?

Reply:
Yep, my bad.